### PR TITLE
Forward RSS feeds requests to observer

### PR DESCRIPTION
**Pull request by giorgiosironi**

### CODE REVIEW

File: salt/pillar/journal.sls [hunk 1]
```diff
@@ -3,6 +3,7 @@ journal:
     api_url_public: '%api_url%'
     api_key: key_for_authorizing_api_requests
     side_by_side_view_url: https://lens.elifesciences.org/
+    observer_url: https://prod--observer.elifesciences.org/
```
Comment (thewilkybarkid) [inline salt/pillar/journal.sls]: Avoid `prod--` here?

Comment (giorgiosironi) [inline salt/pillar/journal.sls]: Like for Lens, I was pointing to read-only prod as a source of data. I'll just put `null`.

Comment (giorgiosironi) [inline salt/pillar/journal.sls]: Or maybe you meant to point to `observer.elifesciences.org`, I'll do that instead.

**Comment by giorgiosironi** [comment]
Merging to end-to-end test it.

**Pull request merged**